WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -266,18 +275,25 @@ private void PerformUpdate(WtsModel wts, Version currentVersion, Version version
                     return;
                 }
             }
+
+            var subject = "WTS Auto Updater - Update installed";
+            var message = $"WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.";
             
-            logger.LogInformation($"WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.");
+            logger.LogInformation(message);
 
             if (wts.SendEmailOnUpdateComplete)
             {
-                EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Update installed", $"The service for WTS '{wts.ServiceName}' has been successfully updated to version {versionToUpdateTo}.", wts.ServiceName);
+                InformPeople(wts,subject,message);
             }
         }
         catch (Exception e)
         {
+            var subject = "WTS Auto Updater - Updating failed!";
+            var message= $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}.{Environment.NewLine}{Environment.NewLine} Error when updating:<br/>{e}";
+            
             logger.LogError($"Exception occured while updating WTS '{wts.ServiceName}'.{Environment.NewLine}{Environment.NewLine}{e}");
-            EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Updating failed!", $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}.<br/><br/>Error when updating:<br/>{e.ToString().ReplaceLineEndings("<br/>")}", wts.ServiceName);
+            
+            InformPeople(wts,subject,message);
```

Comment:
I'm not going to reject the PR for this, but please separate parameters with spaces.